src/main.rs

```diff
@@ -52,6 +52,14 @@ fn install_panic_handler() {
     }
 }
 
+// Exit with return code 0 on SIGINT and SIGTERM
+fn install_signal_handler() {
+    ctrlc::set_handler(move || {
+        std::process::exit(0);
+    })
+    .expect("Error setting SIGINT and SIGTERM handler");
```

Comment:
Given that all the signal handler itself does is provide a slightly nicer UX, crashing lorri if the handler cannot be set seems excessive. Under what circumstances would setting the handler fail?
- If we expect `set_handler` to fail every now and then, I would propose just logging a warning on failure and moving on.
- If `set_handler` fails "basically never", then calling `expect` here seems fine.

Reply:
Thanks. this makes sense as a concern!

Judging from the code and the variants of the  `ctrlc::Error` [enum](http://detegr.github.io/doc/ctrlc/enum.Error.html) it seems like it `set_handler` can fail  if:
1. The SIGINT or SIGTERM signals don't exist on the underlying system 
2. a programmer error where multiple Ctrl-C handlers have been registered
3. an expected system error (seems like it can only happen when basic Unix operations fail in surprising ways).

So it seems to me that `set_handler` should fail "basically never". 

I'm happy to change it to log a warning on failure if you prefer! But in that case, one concern might be that if it were to fail in that exceptional way and we don't exit the process immediately, `lorri daemon` might not respond to SIGINT or SIGTERM at all (I'm not sure how justified this concern is).

Reply:
> So it seems to me that `set_handler` should fail "basically never".

Thanks for providing this context! Given your analysis, I agree - `expect` seems like the right thing to use here.